src/util/triplea/MapXMLCreator/MapPropertiesPanel.java

```diff
@@ -0,0 +1,360 @@
+		final JTextField tWaterFilter = new JTextField(MapXMLCreator.s_waterFilterString);
+		final JLabel lMapName = new JLabel("Map Name:");
+		final JLabel lMapNameExample = new JLabel("e.g. 'Revised'");
+		final JLabel lMapVersion = new JLabel("Map Version:");
+		final JLabel lMapVersionExample = new JLabel("e.g. '1.2.0.1'");
+		final JLabel lResourceName = new JLabel("Resource Name:");
+		final JLabel lResourceNameExample = new JLabel("e.g. 'PUs'");
+		final JLabel lMapImageFile = new JLabel("Map Image File:");
+		final JButton bSelectMapImageFile = new JButton("Browse");
+		final JLabel lCentersFile = new JLabel("Map Centers File:");
+		final JButton bSelectCentersFile = new JButton("Browse");
+		final JLabel lWaterFilter = new JLabel("Water Territory Filter");
+		final JLabel lWaterFilterExample = new JLabel("e.g. 'Sea Zone'");		
+		
+		GridBagLayout gbl_panel_1 = new GridBagLayout();
+		gbl_panel_1.rowHeights = new int[] { 32, 32, 32, 32, 32, 32, 32, 0 };
+		gbl_panel_1.rowWeights = new double[] { 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, Double.MIN_VALUE };
+		m_stepActionPanel.setLayout(gbl_panel_1);
+		
+		//Map Name
+		GridBagConstraints gbc_lMapName = new GridBagConstraints();
+		gbc_lMapName.insets = new Insets(0, 0, 5, 5);
+		gbc_lMapName.anchor = GridBagConstraints.NORTHEAST;
+		gbc_lMapName.gridheight = 32;
+		gbc_lMapName.gridy = 1;
+		gbc_lMapName.gridx = 0;
+		m_stepActionPanel.add(lMapName, gbc_lMapName);
+
+		tMapName.addFocusListener(new FocusListener() {
+			
+			@Override
+			public void focusLost(FocusEvent e) {
+				MapXMLHelper.s_xmlStrings.put("info_@name", tMapName.getText());
+			}
+			
+			@Override
+			public void focusGained(FocusEvent e) {
+			}
+		});
+		tMapName.setMaximumSize(new Dimension(0, 0));
+		tMapName.setColumns(30);
+		GridBagConstraints gbc_m_tMapName = (GridBagConstraints) gbc_lMapName.clone();
+		gbc_m_tMapName.anchor = GridBagConstraints.NORTH;
+		gbc_m_tMapName.gridx = 1;
+		tMapName.setMaximumSize(tMapName.getPreferredSize());
+		m_stepActionPanel.add(tMapName, gbc_m_tMapName);
+		
+		GridBagConstraints gbc_lMapNameExample = (GridBagConstraints) gbc_lMapName.clone();
+		gbc_lMapNameExample.anchor = GridBagConstraints.NORTHWEST;
+		gbc_lMapNameExample.gridx = 2;
+		m_stepActionPanel.add(lMapNameExample, gbc_lMapNameExample);
+		
+		//Map Version
+		GridBagConstraints gbc_lMapVersion = (GridBagConstraints) gbc_lMapName.clone();
+		gbc_lMapVersion.gridy = 2;
+		m_stepActionPanel.add(lMapVersion, gbc_lMapVersion);
+
+		tMapVersion.addFocusListener(new FocusListener() {
+			
+			@Override
+			public void focusLost(FocusEvent e) {
+				MapXMLHelper.s_xmlStrings.put("info_@version", tMapVersion.getText());
+			}
+			
+			@Override
+			public void focusGained(FocusEvent e) {
+			}
+		});
+		tMapVersion.setMaximumSize(new Dimension(0, 0));
+		tMapVersion.setColumns(30);
+		GridBagConstraints gbc_m_tMapVersion = (GridBagConstraints) gbc_lMapName.clone();

```

Comment:
The next few chunks of code are duplicates. Some helper functions to help create teh GridBagContraints would help simplify things.